views/fe-utils/fetchUtil.js

```diff
@@ -214,6 +214,18 @@ const fetchGraphTableCSV = async (chart_id, filters, filename) => {
   return res
 }
 
+const updateUserAccess = async (user) => {
+  const res = await window.fetch(apiRoutes.admin(uid), {
+    ...defaultApiOptions,
+    method: 'PATCH',
+    body: user,
+  })
+
+  if (res.status !== 200) return new Error(res.message)
```

Comment:
should this throw the error instead of returning it?

Reply:
4c885b5249ccbc5570f1e915cc0ac3d7d6147536